contracts/utils/cryptography/P256.sol

```diff
@@ -239,7 +239,7 @@ library P256 {
     }
 
     /**
-     * @dev Compute P·u1 + Q·u2 using the precomputed points for P and Q (see {_preComputeJacobianPoints}).
+     * @dev Compute G·u1 + Q·u2 using the precomputed points for G and Q (see {_preComputeJacobianPoints}).
```

Comment:
`_preComputeJacobianPoints` is expressed in terms of `p` / `px` / `py` (the point provided as parameter) and `g` / `GX` / `GY`, the generator for the curve.

After checking, it looks like this function returns `G·u1 + P·u2`, with G the generator and P the point used to build the table.

Reply:
```suggestion
     * @dev Compute G·u1 + P·u2 using the precomputed points for G and P (see {_preComputeJacobianPoints}).
```